vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -66,4 +81,79 @@ public IExhibition getStartExhibition() {
         }
         return null;
     }
+
+    /**
+     * Updates the Exhibition with given list of languages. It fetches the language from exhibitionLanguageConfig using code.
+     *  
+     * @param exhibition
+     * @param defaultLanguage 
+     * @param languages
+     * @throws LanguageListConfigurationNotFound 
+     */
+    @Override
+    public void updateExhibitionLanguages(Exhibition exhibition, List<String> codes, String defaultLanguage) throws LanguageListConfigurationNotFound {
+        if(CollectionUtils.isNotEmpty(codes)) {
+            
+            // Adds defaultLanguage to codes list if not already exists.
+            if(defaultLanguage!=null && !codes.contains(defaultLanguage)) {
+                codes.add(defaultLanguage);
+            }
+
+
+            if(CollectionUtils.isNotEmpty(exhibitionLanguageConfig.getExhibitionLanguageList())) {
+                exhibitionLanguageConfig.getExhibitionLanguageList().stream()
+                    .filter(languageConfig -> codes.contains(languageConfig.get(ConfigConstants.CODE)))
+                    .forEach(languageMap -> {
+                        
+                        ExhibitionLanguage exhibitionLanguage =  addExhibitionLanguage(exhibition , languageMap);                     
+                        updateDefault(exhibitionLanguage, defaultLanguage);
+
+                    });  
+
+            } else {
+                throw new LanguageListConfigurationNotFound("Exhibition Language Configuration not found");
+            }
+        } 
+
+    }
+
+    /**
+     * Adds exhibitionLanguage to exhibition if not already present. If already present, returns exhibitionLanguage from the exhibition.
+     * 
+     * @param exhibition
+     * @param languageMap
+     * @return
+     */
+    private ExhibitionLanguage addExhibitionLanguage(Exhibition exhibition, Map languageMap) {
+        ExhibitionLanguage exhibitionLanguage =   new ExhibitionLanguage((String) languageMap.get(ConfigConstants.LABEL),
+                (String) languageMap.get(ConfigConstants.CODE), exhibition);
+
+        int index =  exhibition.getLanguages().indexOf(exhibitionLanguage);
+        if( index < 0 ) {
+            exhibition.getLanguages().add(exhibitionLanguage);
+        } 
+        else {
+            exhibitionLanguage =   exhibition.getLanguages().get(index);
+
+        }
+
+        return exhibitionLanguage;
+    }
+
+    /**
+     * Updates isDefault to true if equal to defaultLanguage. Otherwise sets to false.
+     * 
+     * @param exhibitionLanguage
+     * @param defaultLanguage
+     */
+    private void updateDefault(ExhibitionLanguage exhibitionLanguage, String defaultLanguage) {
+       
+        if(exhibitionLanguage.getCode().equalsIgnoreCase(defaultLanguage)) {
+            exhibitionLanguage.setDefault(true);
+        } else {
+            exhibitionLanguage.setDefault(false);
+        }
```

Comment:
I don't think you need an extra method for this. Just make it 
`exhibitionLanguage.setDefault(exhibitionLanguage.getCode().equalsIgnoreCase(defaultLanguage))`